src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -52,10 +51,6 @@ public void BindApplicationEvents(Application application)
                 BindVisualElementEvents(visualElement);
             }
 
-            // We'll use reflection to attach to other events
-            // This allows us to attach to events from custom controls
-            BindReflectedEvents(e.Element);
```

Comment:
Is this implicitly a breaking change? Previously people would be getting automatic breadcrumbs but now not? Should it go in `4.0.0`?

Reply:
I would not consider automatically created breadcrumbs a breaking change.